config/locales/de/views/users.yml

```diff
@@ -4,6 +4,23 @@ de:
     devise_mailer:
       confirmation_instructions:
         p1: 'Vielen Dank für Ihre Registrierung bei CodeHarbor! Klicken Sie auf den folgenden Link, um Ihren Account zu bestätigen:'
+    nbp_wallet:
+      connect:
+        cancel_registration: Registrierung abbrechen
+        header: Wallet verbinden
+        info_html: |
+          Um Ihre Registrierung abzuschließen, scannen Sie bitte den QR Code mit Ihrer 'Mein Bildungsraum: Wallet' App. Die 'Mein Bildungsraum: Wallet' App gibt es im <a href='%{app_store_link}' target='_blank' rel='noopener'>App Store</a> und <a href='%{play_store_link}' target='_blank' rel='noopener'>Play Store</a>.
+          <br>
+          Wenn Sie die App auf dem Gerät installiert haben, mit dem Sie diese Seite ansehen, können Sie stattdessen auch <a href='%{alternative_link}'>hier klicken</a>.
+          <br><br>
+          Bitte geben Sie in der App entweder "Lehrer:in" oder "Schüler:in" als Ihre Rolle ein.
```

Comment:
"geben Sie ein" vs. "wählen Sie aus". What is correct/intended? Does the user really have to enter "Lehrer:in" manually?

Having found `lib/enmeshed/status_group_synonyms.yml` now, I fear that "geben Sie ein" is correct here. 🙈

Reply:
Unfortunately, "entering" ("geben Sie ein") is the correct wording. We are dealing with a free-text form entry 🙈 

Having that said, I would tend to keep the current wording for both, German and English. Are you okay with that?

<details>
<summary>Screenshot</summary>

![2 ausgefülltes Formular](https://github.com/openHPI/codeharbor/assets/7300329/dcd90d8f-21d2-4726-aebe-195d35e86840)


</details>